sympy/crypto/rs_func.py

```diff
@@ -0,0 +1,45 @@
+message = [[1,2,3,0],
+          [4,5,6,45],
+          [7,8,9,83]]
+def create(k,l,m,n,c):
+    l1,l2,l3,l4=[],[],[],[]
+    for i in range(k,n-1):
+            l1.append(message[k][i])   #creates a list from 1 to 3
+            message[k][i]=None
```

Comment:
Why the original message is modified?

Reply:
While looping over the message to create lists l1-l4, some elements will be repeated according to this code. To avoid these repeatations I assigned None to the already appended elements so that I can filter out all 'Nones' later and give proper output.
Talking about the repeatations, present code without assigning None does the following;
Assume message==>1 2 3 4
                                   5 6 7 8
                                   9 0 3 4
On first iteration: l1=[1,2,3]; l2=[4,8]; l3=[4,3,0] ; l4=[9,5]
On second iteration:l1=[6,7]; l2=[8]; l3=[4,3]; l4=[0]
As we see some elements are repeated. Also I need to loop over till last column and row, because when I tried to reduce the upper limit, for inner most elements the flow of control never the loop as start and end point was coinciding(mainly because python excludes upper limit).